source/classes.tex

```diff
@@ -6785,10 +6785,76 @@
 \rSec2[class.spaceship]{Three-way comparison}
 \indextext{operator!three-way comparison!defaulted}%
 
+\pnum
+The synthesized three-way comparison
```

Comment:
Please add the index entry `\indextext{operator!three-way comparison!synthesized}`.  FWIW, I would like to see "synthesized three-way comparison" kept as a definition as suggested in the paper - I realize we don't tend to do that when the term is only used locally, but what happens when it starts getting used elsewhere?

Reply:
Done.